client/models/lab_tests_model.ts

```diff
@@ -170,7 +176,35 @@ export interface LabResultsMetadata {
 
 export interface LabResultsResponse {
   metadata: LabResultsMetadata;
-  results: Object;
+  results: Result[];
```

Comment:
I don’t think you can do this because the results might be in the spiriplex format. Wyndly doesn’t use this client but just an FYI

Reply:
Yea I decided to ignore the Wyndly case for now, and assume all results are like this.